addon/components/sortable-item.js

```diff
@@ -148,6 +148,9 @@ export default Component.extend({
   */
   _direction: computed.readOnly('direction'),
 
+
+  disableCheckScrollBounds: Ember.testing,
```

Comment:
Ember.testing should not be used. See discussion here: https://github.com/emberjs/rfcs/pull/619#issuecomment-618789784

Reply:
I can not use the solution you suggest with `import config from 'my-application/config/environment';` in an add-on :thinking: What do you suggest here ? Using `ember-get-config`?

Reply:
I think that would be fine.

Reply:
OK, it's done.